pkg/runner/cluster_k8s.go

```diff
@@ -251,7 +237,15 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.W
 			eg.Go(func() error {
 				defer func() { <-sem }()
 
-				return c.createPod(ctx, podName, input, runenv, env, g, i)
+				currentEnv := make([]v1.EnvVar, len(env))
+				copy(currentEnv, env)
+
+				currentEnv = append(currentEnv, v1.EnvVar{
+					Name:  "TEST_OUTPUTS_PATH",
+					Value: fmt.Sprintf("/outputs/%s/%s/%d", input.RunID, g.ID, i),
```

Comment:
@raulk note that we no longer have the `testplan name` (for example `tg-dht`) in the `/outputs` path.

The reason for the change is that if a run is canceled or fails, and a user calls `testground collect --runner cluster:k8s 920588cc18f3`, we have no idea in which dir to look for `920588cc18f3`. In the current scheme, we check:
1. Is a `tgz` file present? If yes - then the run must have succeeded (or someone already called `testground collect`)
2. If a `tgz` is not present, it means that the run was not successful, so we archive the outputs at that point - whatever has been stored...

I hope that makes sense.